pennylane_lightning/core/_serialize.py

```diff
@@ -68,6 +68,14 @@ def __init__(self, device_name, use_csingle: bool = False):
                     f"Pre-compiled binaries for {device_name}"
                     " serialize functionality are not available."
                 ) from exception
+        elif device_name == "lightning.gpu":
+            try:
+                import pennylane_lightning.lightning_gpu_ops as lightning_ops
+            except ImportError as exception:
+                raise ImportError(
+                    f"Pre-compiled binaries for {device_name}"
+                    " serialize functionality are not available."
```

Comment:
This string reads a little weird to me. Do we want to say:
```
f"Pre-compiled binaries for {device_name}"
"are not available."
```
?

Reply:
yea, good spot. Same issue for other devices. Let me update all.